overlord/ifacestate/udevmonitor/udevmon_test.go

```diff
@@ -60,6 +61,15 @@ func (s *udevMonitorSuite) TestDiscovery(c *C) {
 		remInfo = inf
 		callbackChannel <- struct{}{}
 	}
+	done := func() {
```

Comment:
(nitpick) would be nice to call this "enumerateDone" - when I looked at this I though we could change the code here to only look at done but this test also simulates hotplugs after the enumeration was done. So a slightly different name would probably have given me the extra context :)